data_preprocessing/pandas_data_preprocessor.py

```diff
@@ -40,6 +41,14 @@ def save_data(self, folder_path):
     def transform_column_data_to_logarithmic_scale(self, column: str):
         self.data[column] = np.log(self.data[column] + 1)
 
+    def scaling_column_data_numerical_attributes(self, column: str):
+        scaler = StandardScaler()
+        self.data[column] = scaler.fit_transform(self.data[column])
+
+    def encoding_column_data_categorical_attributes(self, column: str):
+        label_encoder = LabelEncoder()
+        self.data[column] = label_encoder.fit_transform(self.data[column])
```

Comment:
[Sugestion] Probably `StandardScaler` and `LabelEncoder` could be initialize once in __init__ method

```python
    def __init__(self, data_type='general'):
        super().__init__()
        logging.basicConfig(level=logging.INFO,
                            format='%(asctime)s - %(name)s - %(levelname)s - %(data_type)s - %(message)s')
        self.logger = logging.getLogger(__name__)
        self.data_type = data_type
        self.scaler = StandardScaler()
        self.label_encoder = LabelEncoder()
```

Reply:
Good point, our code will be better organized